Documentation/workflow/UsingCoreRun.md

```diff
@@ -0,0 +1,81 @@
+
+# Using CoreRun To Run .NET Core Application
+
+In page [Using Your Build](UsingYourBuild.md) gives detailed instructions on using the standard
+command line host and SDK, dotnet.exe to run a .NET application with the modified build of the
+.NET Core runtime built here.   This is the preferred mechanism for you to officially deploy 
+your changes to other people since dotnet.exe and Nuget insure that you end up with a consistent
+set of DLLs that can work together.  
+
+However packing and unpacking the runtime DLLs adds extra steps to the deployment process and when 
+you are in the tight code-build-debug loop these extra steps are an issue.   
+
+For this situation there is an alternative host to dotnet.exe called coreRun.exe that is well suited
+for this.   It does not know about Nuget at all, and has very simple rules.  It needs to find the
+.NET Core runtnime (that is coreclr.dll) and additionally any class library DLLs (e.g. System.Runtime.dll  System.IO.dll ...).
+
+It does this by looking at two environment variables.   
+
+
+ * CORE_ROOT - The directory where to find the runtime DLLs itself (e.g. CoreCLR.dll).   
+ Defaults to be next to the coreRun.exe host itself.  
```

Comment:
coreRun casing should be either CoreRun or corerun. corerun is more Unix friendly because of it is the actual casing on Unix and Unix is case sensitive